tests/unit/cartography/graph/test_cleanupbuilder.py

```diff
@@ -120,14 +122,49 @@ def test_get_params_from_queries():
     assert set(get_parameters(queries)) == {'UPDATE_TAG', 'sub_resource_id', 'LIMIT_SIZE'}
 
 
-def test_build_cleanup_queries_selected_rels_no_sub_res_raises_exc():
-    """
-    Test that not specifying the sub resource rel as a selected_relationship in build_cleanup_queries raises exception
-    """
-    with pytest.raises(ValueError, match='node_schema without a sub resource relationship is not supported'):
-        build_cleanup_queries(SimpleNodeSchema())
-
-
 def test_build_cleanup_node_and_rel_queries_sub_res_tgm_not_validated_raises_exc():
     with pytest.raises(ValueError, match='must have set_in_kwargs=True'):
         _build_cleanup_node_and_rel_queries(FakeEC2InstanceSchema(), FakeEC2InstanceToAWSAccount())
+
+
+def test_build_cleanup_queries_no_sub_resource():
+    # Ensure that we only clean up stale relationships for node schemas that don't have a sub resource defined.
+    # That is, we do not delete stale nodes.
+    actual_queries: list[str] = build_cleanup_queries(GitHubOrganizationUserSchema())
+    expected_queries = [
+        '''
+        MATCH (n:GitHubUser)
+        MATCH (n)-[r:MEMBER_OF]->(:GitHubOrganization)
+        WHERE r.lastupdated <> $UPDATE_TAG
+        WITH r LIMIT $LIMIT_SIZE
+        DELETE r;
+        ''',
+        '''
+        MATCH (n:GitHubUser)
+        MATCH (n)-[r:ADMIN_OF]->(:GitHubOrganization)
+        WHERE r.lastupdated <> $UPDATE_TAG
+        WITH r LIMIT $LIMIT_SIZE
+        DELETE r;
+        ''',
+    ]
+    assert clean_query_list(actual_queries) == clean_query_list(expected_queries)
```

Comment:
nit: fodder for a different PR - might be better to compare sets here instead of lists